### PR TITLE
KAA-1160: Glossary - fixed upon AK review.

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

File: doc/Glossary/index.md [hunk 1]
```diff
@@ -5,74 +5,198 @@ permalink: /:path/
...
-| U                                                       |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
-| User verifier                                           | A server component that handles the user verification. Refer to User verifiers for more information.
```
Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: A virtual entity that operates within the [actor gateway]({{root_url}}Glossary/#actor-gateway) to impersonate a physical device, virtual object, library, etc. as a Kaa [endpoint]({{root_url}}Glossary/#endpoint).

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: A Kaa-based component that initializes, manages, and uses [actors]({{root_url}}Glossary/#actor) to represent endpoints connected to the gateway.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: Remove this. It's not only available via Sandbox.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: Deployments with an actor gateway are intended for IoT environments that use proximity or non-Internet communication protocols, and thus have to be bridged to communicate to the server.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: A certain version of a [Kaa application]({{root_url}}Glossary/#kaa-application) whose behavior is defined by a specific set of extensions and their configurations.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: You can copy Application token from the main window of the [Administration UI]({{root_url}}Glossary/#administration-ui).

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: Avro UI forms are used to create Kaa data schemas and data records in the visual environment as opposed to JSON syntax.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: This link is wrong as this content should not be there is the first place.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: Kaa clients use a Bootstrap services list for a certain [Kaa instance]({{root_url}}Glossary/#kaa-instance-kaa-deployment) embedded in the SDK.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: This is misleading (wrong instance term reference etc.). Remove.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: A versioned data schema structure definition, managed within the [Common type library]({{root_url}}Glossary/#common-type-library-ctl).

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: Also see [Common type library]({{root_url}}Programming-guide/Key-platform-features/Common-type-library) documentation section.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: See [Common type library]({{root_url}}Programming-guide/Key-platform-features/Common-type-library) documentation section.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: A repository of [common types]({{root_url}}Glossary/#common-type-ct) that can be reused in Kaa modules.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: remove

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: remove

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: Wrong. Remove.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: Replace with:
Kaa represents every managed entity: device, sensor, mobile phone, etc. as an endpoint.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: Replace with:
An endpoint group is defined by a set of filters against the endpoint profile.
Those endpoints, profiles of which match the filter, get automatically assigned to a group.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: An isolated set of configurations and code that utilize [Kaa platform]({{root_url}}Glossary/#kaa-platform) functionality for a specific use case.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: Kaa client typically uses Kaa [endpoint SDK]({{root_url}}Glossary/#endpoint-sdk) to communicate to the [Kaa server]({{root_url}}Glossary/#kaa-server).

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: An independent software module that implements an isolated set of functions aimed at extending the communication capabilities between Kaa server and Kaa clients.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: remove

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: The Kaa platform comprises [Kaa server]({{root_url}}Glossary/#kaa-server) and [endpoint SDKs]({{root_url}}Glossary/#endpoint-sdk).

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: Currently, there are four SDK types supported in Kaa: Java, C, C++, Objective-C.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: A particular way of rebalancing the workload between the [Operations services]({{root_url}}Glossary/#operations-service) within a [Kaa cluster]({{root_url}}Glossary/#kaa-cluster) to achieve the desired load for each service.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: Wrong. It can actually use multiple. Remove.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: You can [create custom log appenders]({{root_url}}Customization-guide/Customizable-system-components/Log-appenders).

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: remove

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: The operation service is mainly intended for data exchange and synchronization with [endpoints]({{root_url}}Glossary/#endpoint-ep).

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: remove

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: remove

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: A predicate expression based on the Spring Expression Language designed to filter endpoint profiles.
By applying profile filters Kaa platform identifies which endpoints should be attributed to a specific endpoint group.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: remove

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: A business entity that uses a part or a whole [Kaa instance]({{root_url}}Glossary/#kaa-instance-kaa-deployment) to independently run [Kaa applications]({{root_url}}Glossary/#kaa-application).

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: A set of specific [Kaa platform]({{root_url}}Glossary/#kaa-platform) functionality that is logically integrated and managed as a single system.

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: A component that implements communication between clients and [Kaa server]({{root_url}}Glossary/#kaa-server).

**Review by akokhanovskyi — CHANGES_REQUESTED** [review]

**Review by akokhanovskyi — APPROVED** [review]

**Comment by hunch7** [comment]
Fixed as per the comments.

**Review by akokhanovskyi — APPROVED** [review]

**Pull request merged**